### ISSUE TITLE
"lbrynet file list --sort=channel_name" crashes when a file has no channel

### ISSUE THREAD

**Issue by gardenappl** [lbryio/lbry-sdk#3240]
As the title suggests, if there's a file with no channel associated with it:
```json
{
  "code": -32500,
  "data": {
    "args": [],
    "command": "file_list",
    "kwargs": {
      "reverse": false,
      "sort": "channel_name"
    },
    "name": "TypeError",
    "traceback": [
      "Traceback (most recent call last):",
      "  File \"lbry/extras/daemon/daemon.py\", line 709, in _process_rpc_call",
      "  File \"lbry/extras/daemon/daemon.py\", line 2023, in jsonrpc_file_list",
      "  File \"lbry/file/file_manager.py\", line 296, in get_filtered",
      "  File \"lbry/file/file_manager.py\", line 296, in <genexpr>",
      "  File \"lbry/file/source_manager.py\", line 135, in get_filtered",
      "TypeError: '<' not supported between instances of 'NoneType' and 'str'",
      ""
    ]
  },
  "message": "'<' not supported between instances of 'NoneType' and 'str'"
}```

**Comment by tzarebczan** [comment]
Nice catch and thanks for filing!

**Comment by lyoshenka** [comment]
fixed in #3303
